config/env/production.json

```diff
@@ -3,9 +3,9 @@
         "name": "Cards for Humanity"
     },
     "facebook": {
-        "clientID": "APP_ID",
-        "clientSecret": "APP_SECRET",
-        "callbackURL": "http://cfh.io/auth/facebook/callback"
+        "clientID": "180680672583006",
+        "clientSecret": "3cfd8dd9cb1116aedc701bd2138b66d2",
```

Comment:
This is sensitive information. The secret token should not be included in the code. Use a dotenv file to hold this information instead.